cmd/titus-log-viewer/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/Netflix/titus-executor/darion/api"
+	"github.com/Netflix/titus-executor/logsutil"
+	log "github.com/sirupsen/logrus"
+)
+
+func pingHandler(w http.ResponseWriter, r *http.Request) {
+	if _, err := w.Write([]byte("pong")); err != nil {
+		log.Error("Unable to respond with pong on ping handler: ", err)
+	}
+}
+
+func main() {
+	logsutil.MaybeSetupLoggerIfUnderSystemd()
+	log.Println("Titus log viewer is starting")
+	r := newMux()
+	if err := http.ListenAndServe(":8004", r); err != nil {
```

Comment:
Is this meant to run in a different network namespace where it won't clash on port with Darion?

Reply:
yes